dobi.yaml

```diff
@@ -0,0 +1,93 @@
+
+meta:
```

Comment:
Agreed on the compose point.
Interesting point about the docker cache, and yes that makes perfect sense, thank you!
SSH... yes, that's an issue that has started to annoy me as well. I might actually solve it separately, it's no big deal. If you don't want to wait for me, I suggest a separate PR.